v2/config/samples/compute/v1beta20220301_virtualmachinescaleset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: compute.azure.com/v1beta20220301
+kind: VirtualMachineScaleSet
+metadata:
+  name: samplevmss
+  namespace: default
+spec:
+  location: westcentralus
+  owner:
+    name: aso-sample-rg
+  sku:
+    name: standard_d1_v2
```

Comment:
Not sure if this would go through in lower case.. Ideally should be ``STANDARD_D1_v2`

Reply:
That's the casing that comes back from the service, so I'm pretty sure it works.

Reply:
I think they're case insensitive on this one (it's not an enum like many other fields, it's just a raw string)